spec/spec-helper.coffee

```diff
@@ -51,7 +51,7 @@ Object.defineProperty document, 'title',
 
 jasmine.getEnv().addEqualityTester(_.isEqual) # Use underscore's definition of equality for toEqual assertions
 
-if process.env.JANKY_SHA1 and process.platform is 'win32'
+if (process.env.JANKY_SHA1 and process.platform is 'win32') or process.env.CI
```

Comment:
The windows/janky_sha1 check can actually just be completely removed here since the specs no longer run on Windows on janky.